Tests/configure_and_test_integration_instances.py

```diff
@@ -120,8 +121,8 @@ def check_test_version_compatible_with_server(test, server_version, prints_manag
     """
     test_from_version = test.get('fromversion', '0.0.0')
     test_to_version = test.get('toversion', '99.99.99')
-    if (server_version_compare(test_from_version, server_version) > 0
-            or server_version_compare(test_to_version, server_version) < 0):
+    if (LooseVersion(test_from_version) > LooseVersion(server_version)
```

Comment:
Shouldn't this be inclusive?
```suggestion
    if (LooseVersion(test_from_version) >= LooseVersion(server_version)
```